e2e/src/usecases/DappListDisplay.js

```diff
@@ -40,7 +40,9 @@ export default DappListDisplay = () => {
   })
 
   it('should show dapp bottom sheet when dapp is selected', async () => {
-    await element(by.id('DappCard')).atIndex(dappToTest.index).tap()
+    await sleep(2000)
```

Comment:
Is this needed? Is this enough?
Sounds like a potential situation of a flaky test.